app/javascript/scavenger_hunt/scenes/Questions/QuestionOne/QuestionOne.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
We don't need this import since the component has no props.

Reply:
Oh yeah, true. Will remove it Thanks